rust/libits-client/src/pipeline.rs

```diff
@@ -162,23 +164,42 @@ fn mqtt_router_dispatch_thread(
             //initialize the router
             let router = &mut mqtt_router::Router::new();
 
+            // FIXME Iterate on topic list instead of c/p the same code
+
```

Comment:
useless empty line

Reply:
I will actually push the fixed version and remove the FIXME